### DISCUSSION TITLE
Release schedule for upcoming versions

### DISCUSSION THREAD

**Discussion by smartexpert** [duckdb/duckdb#6200] [General]
Hey guys,

When is the new release planned for duckdb? Is there a roadmap or release schedule publicly available?

I want to use a feature that's been merged in dev but not yet released.

Also, the Python version of the pre-release is not installable using poetry

![image](https://user-images.githubusercontent.com/1627433/218130958-2feaeff5-53ac-4aab-9736-212f4359e6f7.png)

Any pointers on how I can sort this out will be appreciated.

**Comment by Mause** [comment]
We expect the next release to be next week sometime

With regards to that error, we don't upload all wheels at the same time for dev builds, so you might have to install a slightly older dev build

**Reply by smartexpert** [reply]
That's great! Thank you!

 Is there a roadmap or release schedule publicly available
